doc/ansible/dev/retroactively-owned-resources.md

```diff
@@ -0,0 +1,162 @@
+# Owner References for Existing Resources
+
+Owner references are automatically injected *only during creation of
+resources*. Enabling owner reference injection *will not update objects*
+created while [owner reference injection is
+disabled](./advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
```

Comment:
```suggestion
Owner references are automatically injected `only during creation of
resources`. Enabling owner reference injection `will not update objects`
created while [owner reference injection is
disabled](./advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)```

The docs layout would be used `` to highlight it instead of ** which will make the text be italic.